test/grouped_convnd_fwd/test_grouped_convnd_fwd_large_cases_xdl.cpp

```diff
@@ -52,8 +52,7 @@ using namespace ck::tensor_layout::convolution;
 
 using KernelTypes2d = ::testing::Types<std::tuple<float, NHWGC, GKYXC, NHWGK>,
                                        std::tuple<ck::half_t, NHWGC, GKYXC, NHWGK>,
-                                       std::tuple<ck::bhalf_t, NHWGC, GKYXC, NHWGK>,
-                                       std::tuple<int8_t, NHWGC, GKYXC, NHWGK>>;
```

Comment:
Why removing int8 ??

Reply:
If this is a bug, please create a ticket for this and just comment out this data type with proper information. It would be best if we would be able to just disable this but I'm not sure if it's possible for parameterized tests.